deploy/playbooks/roles/common/templates/prod_nodes.py.j2

```diff
@@ -45,7 +45,8 @@ nodes = {
     },
     'jessie_small': {
         'script': dedent("""#!/bin/bash
-        wget -q -O - --auth-no-challenge --http-user={{ prado_user | default('admin') }} --http-password={{ prado_token }} "{{ prado_url }}/setup/slave/?token={{ jenkins_prado_token }}&executors=1&labels=amd64+wheezy+small+x86_64&nodename=jessie_small__%s" | bash
+        apt-get update && apt-get install -y -q curl
+        wget -q -O - --auth-no-challenge --http-user={{ prado_user | default('admin') }} --http-password={{ prado_token }} "{{ prado_url }}/setup/slave/?token={{ jenkins_prado_token }}&executors=1&labels=amd64+jessie+small+x86_64&nodename=jessie_small__%s" | bash
```

Comment:
it can't be done in prado, because prado only acts after this has been downloaded. Chicken or the egg: we all know the chicken came first, in this case, the chicken is the cloud-init call, the egg that contains ansible is prado.

Will refactor to go back to LE CURL
![laacw5a](https://cloud.githubusercontent.com/assets/317847/13998094/a72de9c6-f10b-11e5-94a9-abf7da02b3be.gif)